js/src/lib/Details/Aside.js

```diff
@@ -41,6 +43,10 @@ const Aside = (
         >
           {window.i18n.detail.try_in_runkit}
         </a>
+        {' \xB7 '}
```

Comment:
That's fine. What about using the Unicode dot directly instead? No big deal anyway